ddcz/templates/creative-pages/commonarticle-list.html

```diff
@@ -6,7 +6,7 @@
 
 {% block content %}
 <div id="page-common-article">
-    <h1 class="page-heading">{{ heading }}</h1>
+    <h1 class="page-heading {% if heading == 'Články&Eseje' %}text-red{% endif %}">{{ heading }}</h1>
```

Comment:
I'd rely on `slug` more than on heading (also because I think there should be a whitespace around ampersand, plus it should be encoded in HTML